docs/questions/native-editor/writing-sql.md

```diff
@@ -88,13 +88,12 @@ On saved SQL questions without [parameters](./sql-parameters.md), you'll get the
 
 ![Explore results button](../images/explore-results.png)
 
-## To enable drill-through, turn a SQL question into a model and set the data types
+## Drill-though in SQL questions
```

Comment:
Removing the "turn into a model" section because turning into a model only enabled the drill-through options that we now have with native drill-through